.github/pull_request_template.md

```diff
@@ -0,0 +1,36 @@
+## Proposed changes
```

Comment:
I think we can store this file as `CONTRIBUTING.md` in the root directory. It'll then be added to the contributions section of GitHub. We can link to it from the `README.md` file.